gulp/react/spark-core-react/spark-core-react.gulpfile.js

```diff
@@ -41,6 +41,14 @@ gulp.task('link-spark-core-react', (cb) => {
   });
 });
 
+gulp.task('test-spark-core-react', (cb) => {
+  exec('cd src/react/projects/spark-core-react && ../../node_modules/.bin/jest spark-core-react', (err, stdout, stderr) => {
```

Comment:
should run from spark-core-react's copy of jest

Reply:
@rcopeland ah yep I’ll change it! I noticed the Angular is setup this way.

Reply:
Yep, In angular's case, the projects folder is part of the angular project set up in src/angular, there's only one copy of angular installed, so that's why it runs from there.

Reply:
Ahhhh! That makes sense!

Reply:
Should be fixed now.